index.html

```diff
@@ -1,24 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Portfolio Microverse!</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/port.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Mohamed Portfolio</title>
 </head>
 <body>
-    <header>
-        <div>
-            <img src="src/img/logo.png" class="logo" alt="Portfolio">
-            <img src="src/img/menu-button.png" class="menu-button" alt="Menu">
-        </div>
+  <!--header-->
+ <div class="bk_container">
+  <div class="port_container">
+    <header class="header">
+      <nav class="navbar">
+        <a href="#" class="logo">
+          <img src="./images/Portfolio_Logo.png" alt="My Logo"/></a>
+         <a>
+          <img src="./images/Enabled.png" alt="Menu"/>
+        </a>
+      </nav>
     </header>
-    <section class="headline">
-        <h2>I’m Mohamed Glad to see you!</h2>
-        <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <p class="letsconnect">LET’S CONNECT</p>
-        <img src="src/img/social-media-icons.png" class="social-media-icons" alt="Social Media">
+   <!--main section--> 
+    <section id="main_section" class="grid" >
+     <div class="container">
+      <h1 class="h1">I’m Mohamed Baalla Glad to see you!</h1>
```

Comment:
- [ ] Nice to have used `<h1>` here for this is the most important heading you have in the whole Portfolio website. However, let's split the `Glad to see you!` text as a sub-heading from the main heading and wrapping with `<h2>, <h3>, .. <h6>`